ghcide/src/Development/IDE/LSP/LanguageServer.hs

```diff
@@ -94,14 +95,14 @@ runLanguageServer
     -> (MVar ()
         -> IO (LSP.LanguageContextEnv config -> TRequestMessage Method_Initialize -> IO (Either (TResponseError Method_Initialize) (LSP.LanguageContextEnv config, a)),
                LSP.Handlers (m config),
-               (LanguageContextEnv config, a) -> m config <~> IO))
+               (LanguageContextEnv config, a) -> m config <~> IO, [IO ()]))
```

Comment:
```suggestion
               (LanguageContextEnv config, a) -> m config <~> IO, 
               [IO ()]))
```

And since you are already here, you could document the type :)